### PR TITLE
Add relationship links to serialized data

### LINKED ISSUE #14: Addon causes query calls to not make request
I am using this addon in conjunction with the `ember-data-has-many-query` add on to query `hasMany` relationships. When i load the primary model in fastboot but then call `query` on the the has many from the client no request is ever made. If i remove this addon it works as expected. I checked and made sure that `ember-data-has-many-query` is calling `reload` so a request should be made.

### PR DESCRIPTION
**Pull request by arschmitz**
Fixes #14

### CODE REVIEW

**Comment by arschmitz** [comment]
@ef4 would be great to get a review of this while your at its a huge issue for jsonAPI and there is no way i know of in to work around it. Right now i'm forced to use a patched version of the lib

File: fastboot/instance-initializers/ember-data-fastboot.js [hunk 1]
```diff
@@ -9,7 +9,25 @@ export function initialize(applicationInstance) {
...
         .filter(record => record.get('isLoaded'))
-        .map(record => record.serialize({ includeId: true}))
+        .map(record => {
+          const serializedRecord = record.serialize({ includeId: true});
+
+          record.eachRelationship((name, meta) => {
+            const link = record[meta.kind](name).link();
+
+            if (link) {
+              const dashName = name.dasherize();
```
Comment (ef4) [inline fastboot/instance-initializers/ember-data-fastboot.js]: This assumes prototype extensions are turned on, better to use the explicit form like `Ember.String.dasherize(name)`.

**Comment by pichfl** [comment]
@arschmitz any plans on updating your PR? If you're occupied otherwise I can fork your fork and re-file the PR for you.

**Comment by arschmitz** [comment]
@pichfl thank you for pinging me ill update ASAP just totally lost track of this PR

**Comment by arschmitz** [comment]
updated

**Pull request merged**